lib/Twig/Profiler/DumperInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2015 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+interface Twig_Profiler_DumperInterface
+{
+    public function dump(Twig_Profiler_Profile $profile);
```

Comment:
you should document the return type of the method (if the return type cannot be part of the interface, there is no point providing an interface)